easybuild/easyconfigs/f/FLASH/FLASH-2.2.00-foss-2022b.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'FLASH'
+version = '2.2.00'
+
+homepage = 'https://ccb.jhu.edu/software/FLASH/'
+description = """FLASH (Fast Length Adjustment of SHort reads) is a very fast
+and accurate software tool to merge paired-end reads from next-generation
+sequencing experiments. FLASH is designed to merge pairs of reads when the
+original DNA fragments are shorter than twice the length of reads. The
+resulting longer reads can significantly improve genome assemblies. They can
+also improve transcriptome assembly when FLASH is used to merge RNA-seq data.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
```

The Makefile looks rather straightforward, I don't see anything that suggests we need the full `foss` toolchain here. We should be able to drop this to `GCCcore`
```suggestion
toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
```